tests/test_frame.py

```diff
@@ -1389,6 +1389,7 @@ def test_set_columns() -> None:
     df = pd.DataFrame({"a": [1, 2, 3], "b": [0.0, 1, 1]})
     # Next line should work, but it is a mypy bug
     # https://github.com/python/mypy/issues/3004
+    # pyright doesn't need the ignore
```

Comment:
This is almost the only ignore in the tests that is not needed by pyright (there are many in the stubs that are not "needed").

The other two ignores are in `tests/__init__.py`.

If pyright had no unnecessary ignore comments in the tests, we could have a second pyright config for the installed stub check (it only checks the tests using the stubs, but not the stubs).